tests/test_device/test_ipu/test_ipu_utils.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+
+import pytest
+import torch.nn as nn
+
+import mmcv
+from mmcv.device.ipu import IS_IPU
+
+if IS_IPU:
+    from mmcv.device.ipu import cast_to_options
+    from mmcv.device.ipu.utils import model_sharding
```

Comment:
```suggestion
    from mmcv.device import (cast_to_options, model_sharding, build_from_cfg_with_wrapper)
```

Reply:
model_sharding, build_from_cfg_with_wrapper only used in ipu module, so I didn't put them in mmcv/device/ipu/__init__.py

Reply:
got it